ci/run.sh

```diff
@@ -93,15 +96,18 @@ if [ -z "$DEBUG_LTO_BUILD_DOESNT_WORK" ]; then
   RUSTFLAGS="-C debug-assertions=no" \
     CARGO_INCREMENTAL=0 \
     CARGO_PROFILE_DEV_LTO=true \
-    $cargo rustc --features "$INTRINSICS_FEATURES" --target $1 --example intrinsics
+    $cargo rustc --features "$INTRINSICS_FEATURES" --target $1 --example intrinsics --release
```

Comment:
This should not have `--release`: this test specifically checks that *debug* builds (`-Copt-level=0`) do not reference any symbols in core. This is needed for people using `-Z build-std` without `--release`.

Reply:
So I understand that we might expect that only `compiler-builtins` is optimized.